validation/src/test/groovy/io/micronaut/validation/executable/NullablePrimitiveSpec.groovy

```diff
@@ -0,0 +1,83 @@
+package io.micronaut.validation.executable
+
+import io.micronaut.annotation.processing.test.AbstractTypeElementSpec
+import spock.lang.Shared
+
+class NullablePrimitiveSpec extends AbstractTypeElementSpec {
+
+    @Shared
+    String errorMsg = "Primitive types can not be null"
+
+    void 'test @Nullable is not allowed on primitive parameters'() {
+        when:
+        buildTypeElement("""
+package test;
+
+@javax.inject.Singleton
+class Foo {
+    void bar(@javax.annotation.Nullable boolean boolPrimitive) {
```

Comment:
This type of scenario should still compile because Micronaut will not be executing this method

Reply:
I'm sorry James, you are right. I was confused at the beginning and didn't really think about the difference on "user calling it's own code" vs "Micronaut calling user code" (like in a controller). It's fixed now.